msal/broker.py

```diff
@@ -135,6 +139,9 @@ def _get_new_correlation_id():
 def _enable_msa_pt(params):
     params.set_additional_parameter("msal_request_type", "consumer_passthrough")  # PyMsalRuntime 0.8+
 
+def _pass_client_sku(params):
+    params.set_additional_parameter("msal_client_sku", "MSAL.Python")  
+    params.set_additional_parameter("msal_client_ver", __version__) 
```

Comment:
```suggestion
def _build_msal_runtime_auth_params(client, authority):
    params = pymsalruntime.MSALRuntimeAuthParameters(client_id, authority)
    params.set_additional_parameter("msal_client_sku", SKU)
    params.set_additional_parameter("msal_client_ver", __version__)
    return params
```

And then change all the `params = pymsalruntime.MSALRuntimeAuthParameters(client_id, authority)` below into `params = _build_msal_runtime_auth_params(client, authority)`, and remove all the `_pass_client_sku(params)`.

This way, it will be less likely to forget adding sku, should we need to add more Runtime flows in the future.